IQKeyboardManager/IQKeyboardManager.m

```diff
@@ -262,7 +262,7 @@ + (IQKeyboardManager*)sharedManager
 	static dispatch_once_t onceToken;
     dispatch_once(&onceToken, ^{
         
-        kbManager = [[self alloc] init];
```

Comment:
Have you tried compiling @huzhongyin at least one time? Buid is failing here. Please see travis-ci here https://travis-ci.org/hackiftekhar/IQKeyboardManager/jobs/389071345